text-utils/col.c

```diff
@@ -104,19 +104,20 @@ struct line_str {
 	CHAR	*l_line;		/* characters on the line */
 	LINE	*l_prev;		/* previous line */
 	LINE	*l_next;		/* next line */
```

Comment:
It would be nice by a separate patch to remove all the typedefs and LINE and CHAR, and use `struct col_line` and `struct col_char`. The typedef makes sense for simple opaque types (like some numbers etc.), otherwise typedef is evil.

Reply:
Good idea, typedefs are removed in 2446db107019c6cdfc99cdfea245870ea7b19b78.